packages/elements/src/tree-select/index.ts

```diff
@@ -650,8 +650,8 @@ export class TreeSelect extends ComboBox<TreeSelectDataItem> {
   protected override onPopupClosed(): void {
     super.onPopupClosed();
     this.updateMemo();
```

Comment:
I found the `updateMemo()` counting the `memo.selected` and `memo.selectable` incorrectly, it must move to the end of the this function. 😊

Reply:
Not need to call `updateMemo()` here we decide to remove it.